### DISCUSSION TITLE
ndjson support

### DISCUSSION THREAD

**Discussion by verkaro** [noborus/trdsql#221] [Ideas]
Great job.

Do you plan on supporting ndjson?

**Comment by noborus** [comment]
thank you.
See the jsonl entry. https://github.com/noborus/trdsql#jsonl

**Comment by verkaro** [comment]
I just read a comparison of ndjson and jsonl. From what I can understand jsonl is the better choice. Thank you for the reply and design choice. Thank you for trdsql.

**Reply by noborus** [reply]
thank you. Supplement README.md with ndjson in parentheses.

**Reply by verkaro** [reply]
okay. i understand even better now. i think. ;)
